src/lib/Repository/UiRelationService.php

```diff
@@ -62,17 +62,18 @@ public function loadRelations(VersionInfo $versionInfo)
     /**
      * Loads reverse relations for a piece of content.
      * Retrieves reverse relations and then sets the field definition and the content type name.
+     * Returns false if the user doesn't have permission to load reverse relations.
      *
      * @param ContentInfo $contentInfo
      *
-     * @return UiRelation[]
+     * @return UiRelation[]|bool
      */
     public function loadReverseRelations(ContentInfo $contentInfo)
     {
         try {
             $reverseRelations = $this->contentService->loadReverseRelations($contentInfo);
         } catch (UnauthorizedException $e) {
```

Comment:
I'd really prefer if we tested explicitly.

In addition, a good remark from @dpobel : `false` is a bit of a magic value. Wrapping it as a constant would make sense here, like `UiRelationService::NO_REVERSE_RELATIONS_PERMISSION` (set to `false`).

Then we can either test it in the template (I find the syntax quite ugly), or test it in the controller, and assign a specific variable instead of `reverseRelations` that we test instead of `same as (false)`.